bootstrap/bootstrap/bootstrap.py

```diff
@@ -74,8 +124,7 @@ def start_core(self) -> None:
 
         print("Attempting to pull an updated image... This might take a while...")
         try:
-            for line in self.low_level_api.pull(f"{image}:{core_version}", stream=True, decode=True):
-                print(line["status"])
+            self.pull_core()
```

Comment:
So the json configuration is not used anymore ?

Reply:
hah, that explains some issues I was having here... updated.